src/Microsoft.HttpRepl.Tests/Commands/ListCommandTests.cs

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.HttpRepl.Commands;
+using Microsoft.HttpRepl.Fakes;
+using Microsoft.HttpRepl.Preferences;
+using Microsoft.Repl.Parsing;
+using Xunit;
+
+namespace Microsoft.HttpRepl.Tests.Commands
+{
+    public class ListCommandTests : CommandTestsBase
+    {
+        [Fact]
+        public async Task ExecuteAsync_NoBaseAddress_ShowsWarning()
```

Comment:
Does `..._NoBaseAddressInSwaggerFile_...` represent the scenario better?  It was a little confusing at first how it fit together until I read the other tests.

Reply:
There's two conditions that cause `ls`/`dir` to not be useful - one is if there is no base address set on HttpState. The other is if there's no swagger file/structure. This test is checking the former. The next test checks the latter. More accurate would be `..._NoBaseAddressOnHttpState_...` I guess, so I'll change it to that.